jwst/resample/resample_step.py

```diff
@@ -79,14 +79,15 @@ def process(self, input):
         #  Get drizzle parameters reference file, if there is one
         if 'drizpars' in self.reference_file_types:
             ref_filename = self.get_reference_file(input_models[0], 'drizpars')
-            self.log.info('Using drizpars reference file: {}'.format(ref_filename))
-            kwargs = self.get_drizpars(ref_filename, input_models)
         else:  # no drizpars reference file found
             ref_filename = 'N/A'
 
         if ref_filename == 'N/A':
             self.log.info('No drizpars reference file found.')
             kwargs = self._set_spec_defaults()
+        else:
+        	self.log.info('Using drizpars reference file: {}'.format(ref_filename))
```

Comment:
The indent error here (too much indent) is causing code compilation to fail.

Reply:
Since everything else appears to be correct, I'm going to go ahead and fix the error myself right in the PR branch, so that tests can be run.